src/assets.js

```diff
@@ -134,63 +134,64 @@ function buildAssetHTML(assetUrl, categories) {
-        <a-mixin id="right" atlas-uvs="column: 3; row: 2"></a-mixin>
-        <a-mixin id="left" atlas-uvs="column: 3; row: 3"></a-mixin>
-        <a-mixin id="both" atlas-uvs="column: 2; row: 1"></a-mixin>
-        <a-mixin id="all" atlas-uvs="column: 3; row: 1"></a-mixin>
-        <a-mixin id="left-straight" atlas-uvs="column: 2; row: 3"></a-mixin>
-        <a-mixin id="right-straight" atlas-uvs="column: 2; row: 2"></a-mixin>
-        <a-mixin id="straight" atlas-uvs="column: 2; row: 4"></a-mixin>
-        <a-mixin id="sharrow" atlas-uvs="totalRows: 4; totalColumns: 8; column: 2; row: 3" scale="1.5 3 1"></a-mixin>
-        <a-mixin id="bike-arrow" atlas-uvs="totalRows: 2; totalColumns: 8; column: 1; row: 2" scale="1 4 1"></a-mixin>
-        <a-mixin id="word-bus" atlas-uvs="totalRows: 8; totalColumns: 8; column: 1; row: 4" scale="3 3 3"></a-mixin>
-        <a-mixin id="word-lane" atlas-uvs="totalRows: 8; totalColumns: 8; column: 2; row: 4" scale="3 3 3"></a-mixin>
-        <a-mixin id="word-taxi" atlas-uvs="totalRows: 8; totalColumns: 8; column: 1; row: 3" scale="3 3 3"></a-mixin>
-        <a-mixin id="word-only" atlas-uvs="totalRows: 8; totalColumns: 8; column: 2; row: 3" scale="3 3 3"></a-mixin>
-        <a-mixin id="word-only-small" atlas-uvs="totalRows: 8; totalColumns: 8; column: 2; row: 3" scale="2.5 2 2.5"></a-mixin>
-        <a-mixin id="word-yield" atlas-uvs="totalRows: 8; totalColumns: 8; column: 1; row: 2" scale="3 3 3"></a-mixin>
-        <a-mixin id="word-slow" atlas-uvs="totalRows: 8; totalColumns: 8; column: 2; row: 2" scale="3 3 3"></a-mixin>
-        <a-mixin id="word-xing" atlas-uvs="totalRows: 8; totalColumns: 8; column: 1; row: 1" scale="3 3 3"></a-mixin>
-        <a-mixin id="word-stop" atlas-uvs="totalRows: 8; totalColumns: 8; column: 2; row: 1" scale="3 3 3"></a-mixin>
-        <a-mixin id="word-loading-small" atlas-uvs="totalRows: 8; totalColumns: 4; column: 4; row: 1" scale="2.75 1.75 2.75"></a-mixin>
-        <a-mixin id="perpendicular-stalls" atlas-uvs="totalRows: 4; totalColumns: 8; column: 5; row: 4" scale="5 10 5"></a-mixin>
-        <a-mixin id="parking-t" atlas-uvs="totalRows: 8; totalColumns: 16; column: 4; row: 7" scale="1.5 2 2"></a-mixin>
-        <a-mixin id="painted-safety-zone" atlas-uvs="totalRows: 4; totalColumns: 4; column: 4; row: 4" scale="8 8 8"></a-mixin>
-        <a-mixin id="hash-left" atlas-uvs="totalRows: 4; totalColumns: 8; column: 7; row: 2" scale="3 6 3"></a-mixin>
-        <a-mixin id="hash-right" atlas-uvs="totalRows: 4; totalColumns: 8; column: 8; row: 2" scale="3 6 3"></a-mixin>
-        <a-mixin id="hash-chevron" atlas-uvs="totalRows: 4; totalColumns: 4; column: 4; row: 2" scale="3 3 3"></a-mixin>
+        <a-mixin id="stencils"></a-mixin>
+        <a-mixin shadow="cast: false" id="right" atlas-uvs="totalRows: 4; totalColumns: 4; column: 3; row: 2" scale="2 2 2" geometry="primitive: plane;" material="src: #stencils-atlas;alphaTest: 0;transparent:true;"></a-mixin>
```

Comment:
You need `geometry="primitive: plane; skipCache: true` everywhere to be sure to create a separate geometry for each entity, otherwise all your entities are using the same geometry and the atlas-uvs component is changing the uvs of the shared geometry so replacing the image for all entities.

Reply:
@vincentfretin excellent, thank you!